summarize.go

```diff
@@ -108,6 +108,9 @@ func printVal(cfg Config, value reflect.Value, indent string) string {
 	buf := bytes.Buffer{}
 
 	v, t := base(value)
+	if v.Kind() == reflect.Ptr && v.IsNil() {
```

Comment:
I think the linter concerns are secondary, where as functionally it's slightly more correct on 171 in case the switch statement covers more cases in the future. That being said, I'm ok with no change too.

Reply:
I've put it on 171 and disabled the lint.